docker-compose-develop.yml

```
@@ -8,6 +8,10 @@ services:
       - RW_ENV=production
       - PORT=3000
       - REDIS_URL=redis://redis:6379
+    volumes:
+      - .:/usr/src/app
+      # Prevent host node_modules from overwriting node_modules in the container
```

Since the dockerfile moved `node_modules` up a directory, this might better be described as
```suggestion
      # Prevent host node_modules from showing up in the container
```

---

Good point, updated!